### PR TITLE
Add Dragino hotspot HP0D

### PR DESCRIPTION
**Pull request by dragino**

### CODE REVIEW

**Comment by Sophi** [comment]
Hello, if you are trying to make a HIP19 proposal, we have moved to a [new repo here](https://github.com/dewi-alliance/hotspot-manufacturers#readme). I'm going to close this request.

**Review by Dany8921 — APPROVED** [review]

File: src/makers/dragino/index.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import antennas from './antennas'
+import hotspots from './hotspots'
+
+export default {
+  antennas,
+  hotspots,
+  id: 21, // TODO: Get maker id from onboarding server
```
Comment (tyler-whitman) [inline src/makers/dragino/index.ts:7]: You should be using maker id 27

**Review by tyler-whitman — CHANGES_REQUESTED** [review]
Please see the above comments and resolve the merge conflicts with main.

File: src/makers/dragino/index.ts (hunk 1 above)
Comment (dragino) [reply src/makers/dragino/index.ts]: Change to 27 now.

**Comment by dragino** [comment]
Will use a new PR to clean this.

**Pull request closed without merging**